src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private String createReport(String dataFromFile) {
+        String[] split = dataFromFile.split(System.lineSeparator());
```

Comment:
It's better to give more informative names to variables
```suggestion
        String[] linesFromFile = dataFromFile.split(System.lineSeparator());
```